src/marqo/tensor_search/tensor_search.py

```diff
@@ -51,6 +51,7 @@
 from marqo.tensor_search.index_meta_cache import get_cache, get_index_info
```

Comment:
We need to extract the dict keys and add them to the filtering data. See below:
https://github.com/marqo-ai/marqo/blob/mainline/src/marqo/tensor_search/tensor_search.py#L480

Reply:
yes, forget this point after the merge, i will put them in a list.

Reply:
solved